src/components/RoomView/OperativeBoard.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import Card from './Card.jsx';
+import Card from './Card';
 import { useSelector } from 'react-redux';
```

Comment:
converted this file to TS previously, had to update the Card component with TS once i converted Card.jsx to TS